src/components/Button/index.tsx

```diff
@@ -39,8 +39,8 @@ export interface ButtonProps extends AnchorProps {
    */
   buttonStyle?: ButtonStyle;
   /**
-   * Sets the button type. By default button.
-   * @default button
+   * Sets the button type. By default submit.
+   * @default submit
```

Comment:
I would rather remove @default and add the link to spec.
```suggestion
   * Sets the button type. By default "submit".
   * @see [W3C](https://www.w3.org/TR/2011/WD-html5-20110525/the-button-element.html#attr-button-type)
```
It will look like
![image](https://user-images.githubusercontent.com/11414342/93348541-03e21e00-f83f-11ea-8a71-03ca3821142c.png)